framework/src/bcs/ADNodalBC.C

```diff
@@ -44,7 +44,8 @@ ADNodalBCTempl<T>::ADNodalBCTempl(const InputParameters & parameters)
     _current_node(_var.node()),
     _u(_var.adNodalValue()),
     _set_components(
-        {getParam<bool>("set_x_comp"), getParam<bool>("set_y_comp"), getParam<bool>("set_z_comp")})
+        {getParam<bool>("set_x_comp"), getParam<bool>("set_y_comp"), getParam<bool>("set_z_comp")}),
```

Comment:
ugh we should suppress those for normal variables??

Reply:
removed them for non-vector instantiations